### ISSUE TITLE
Improve error handling to avoid "NullPointerException" when getters/setters are forgotten.

### ISSUE THREAD

**Issue by Queatz** [realm/realm-java#668]
Unable to build project using realm due to an NPE during build:

```
23:51:03.024 [ERROR] [org.gradle.BuildExceptionReporter] Caused by: java.lang.NullPointerException
23:51:03.025 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.squareup.javawriter.JavaWriter.beginMethod(JavaWriter.java:401)
23:51:03.025 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.squareup.javawriter.JavaWriter.beginMethod(JavaWriter.java:378)
23:51:03.026 [ERROR] [org.gradle.BuildExceptionReporter] 	at io.realm.processor.RealmProxyClassGenerator.emitAccessors(RealmProxyClassGenerator.java:199)
23:51:03.026 [ERROR] [org.gradle.BuildExceptionReporter] 	at io.realm.processor.RealmProxyClassGenerator.generate(RealmProxyClassGenerator.java:171)
23:51:03.026 [ERROR] [org.gradle.BuildExceptionReporter] 	at io.realm.processor.RealmProcessor.process(RealmProcessor.java:239)
23:51:03.027 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.sun.tools.javac.processing.JavacProcessingEnvironment.callProcessor(JavacProcessingEnvironment.java:793)
23:51:03.027 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:722)
23:51:03.028 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.sun.tools.javac.processing.JavacProcessingEnvironment.access$1700(JavacProcessingEnvironment.java:97)
23:51:03.028 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1029)
23:51:03.029 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1163)
23:51:03.029 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1108)
23:51:03.030 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:824)
23:51:03.030 [ERROR] [org.gradle.BuildExceptionReporter] 	at com.sun.tools.javac.main.Main.compile(Main.java:439)
23:51:03.031 [ERROR] [org.gradle.BuildExceptionReporter] 	... 73 more
```

**Comment by jenzz** [comment]
@chris6647 Thanks for mentioning the issue wrt constants w/o getters, saved me loads of time! :smile:

**Comment by falkorichter** [comment]
+1 for this issue. Why is there no proper error message when the getter and setter are missing? This just cost me a couple of hours...

**Comment by cmelchior** [comment]
This has been released as part of 0.79.0

**Comment by Queatz** [comment]
Awesome!
